repairnator/src/main/java/fr/inria/spirals/repairnator/Launcher.java

```diff
@@ -282,6 +301,15 @@ private void mainProcess() throws IOException {
                 completeWorkspace = scanner.readWorkspaceFromInput(input);
                 buildList = scanner.readBuildFromInput(input);
                 break;
+                
+            case FORBEARS:
+            	buildList = scanner.getListOfPassingBuildsFromProjects(input);
+            	break;
+        }
+
+        if (mode == RepairMode.SLUG || mode == RepairMode.FORBEARS) {
+            GoogleSpreadSheetScannerSerializer scannerSerializer = new GoogleSpreadSheetScannerSerializer(scanner);
+            scannerSerializer.serialize();
         }
```

Comment:
I was checking this. You are right. The right "if" is the one that have two conditions.